test/persistence-hooks.suite.js

```diff
@@ -1816,6 +1916,147 @@ module.exports = function(dataSource, should) {
       });
     });
 
+    describe('PersistedModel.replaceOrCreate', function() {
```

Comment:
Yes, that's correct. Please consider changing the copied test cases so that they would fail if run against `updateOrCreate` instead of `replaceOrCreate`.